region/maridia/outer.json

```diff
@@ -2915,8 +2931,7 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
-                    "canSpringBallJumpMidAir",
-                    "canCrouchJump",
+                    "h_canMaxHeightSpringBallJump",
```

Comment:
I think I'd change the strat to recommend using Global crab to assist with all of the jumps to the top.  You wouldn't need `maxHeight`, `stationaryLateral`, `trickyJump`, and probably not `crouchJump` either.
The only hard part is quickly freezing the above Missiles Crab so that you can use two crabs together.